data/hai/hai missions.txt

```diff
@@ -1808,3 +1846,442 @@ mission "Nanachi Meeting"
 				decline
+
+
+
+mission "Strider 0"
+	landing
+	name "Hull Repair Technology"
+	description "The Hai are working on a drone carrier to counter the Unfettered Solifuge, but the creator of the ship is still developing hull repair to make it more effective in combat. Visit the spaceport on <planet> to help the Hai with this technology when you are able."
+	destination "Hai-home"
+	to offer
+		has "Wanderers Solifuge Recon 3: done"
+	to fail
+		has "Strider 1: offered"
+	to complete
+		never
+
+
+
+mission "Strider 1"
+	landing
+	name "Hull Repair Technology"
+	description "The Hai are working on a drone carrier to counter the Unfettered Solifuge, but the creator of the ship is still developing hull repair to make it more effective in combat. Visit the spaceport on <planet> to help the Hai with this technology when you are able."
+	source "Hai-home"
+	to offer
+		has "Strider 0: active"
+	on offer
+		conversation
+			branch "know both"
+				has "license: Remnant"
+				has "license: Coalition"
+			
+			branch "know coalition"
+				has "license: Coalition"
+			
+			branch "know remnant"
+				has "license: Remnant"
+			
+			`You've returned to <planet>, but you don't have access to any hull repair technology. Explore the galaxy and earn the trust of an alien faction that has hull repair technology before returning.`
+				defer
+			
+			label "know both"
+			`You've returned to <planet> to help the Hai to develop hull repair technology. Torsa's father asked if you knew of any aliens with hull repair technology, and you have met the Coalition, who have outfits capable of repairs hulls, and the Remnant, who have ships that repair themselves. Would you like to tell the elders about one of these groups?`
+			choice
+				`	(Yes.)`
+					goto start
+				`	(Not right now.)`
+					defer
+			
+			label "know coalition"
+			`You've returned to <planet> to help the Hai to develop hull repair technology. Torsa's father asked if you knew of any aliens with hull repair technology, and you have met the Coalition, who have outfits capable of repairs hulls. Would you like to tell the elders about the Coalition?`
+			choice
+				`	(Yes.)`
+					goto start
+				`	(Not right now.)`
+					defer
+			
+			label "know remnant"
+			`You've returned to <planet> to help the Hai to develop hull repair technology. Torsa's father asked if you knew of any aliens with hull repair technology, and you have met the Remnant, who have ships that repair themselves. Would you like to tell the elders about the Remnant?`
+			choice
+				`	(Yes.)`
+				`	(Not right now.)`
+					defer
+			
+			label start
+			`	You inform the elders that you have discovered a group that could help them with creating hull repair technology and they invite you to the council chamber. "Greetings, Captain <last>," one of the elders says to you upon entering the chamber. "What news do you have to bring us on your search for hull repair technology?"`
```

Comment:
```suggestion
			`	You inform the elders that you have discovered a group that could help them with creating hull repair technology, and they invite you to the council chamber. "Greetings, Captain <last>," one of the elders says to you upon entering the chamber. "What news do you have to bring us on your search for hull repair technology?"`
```
All good if you don't want the word change here, but still need the comma or it's grammatically incorrect.

Reply:
I can't accept the changes (I'm not the author), but I think it looks good.